src/Compilers/Core/Portable/Symbols/RefKind.cs

```diff
@@ -49,7 +50,7 @@ internal static string ToArgumentDisplayString(this RefKind kind)
             {
                 case RefKind.Out: return "out";
                 case RefKind.Ref: return "ref";
```

Comment:
Per latest LDM, this should probably say `ref readonly now`

Reply:
Per LDM it can match a ref ore ref readonly parameters. But on the argument side it is a ref. I think returning "ref" is correct. - that is all we know given just the argument.

---
In reply to: [141171113](https://github.com/dotnet/roslyn/pull/22285#discussion_r141171113) [](ancestors = 141171113)

Reply:
I agree. We cannot tell from the argument display which method it will bind to.

---
In reply to: [141179053](https://github.com/dotnet/roslyn/pull/22285#discussion_r141179053) [](ancestors = 141179053,141171113)